README.md

```diff
@@ -21,6 +21,17 @@ aws cloudformation create-stack \
   --parameters <(cat config.json)
 ```
 
+
+Alternately, if you prefer to use this repo, clone it and run the following command to set up things locally and create a remote stack.
+```bash
+make && make create-stack
```

Comment:
This is redundant, just `make create-stack` will do it. By itsself `make` just uses the first target, which is build, which ends up being a dependency of `create-stack` anyway.